model/staff.py

```diff
@@ -1,4 +1,26 @@
+import random
+import shelve
+import string
+
 from person import Person 
+
 class Staff(Person):
+    """ This class creates a new staff and inherits from Person class"""
+
+    persons_data = shelve.open('data_files/persons')
```

Comment:
What will be the difference between putting this as a class variable as opposed to using as instance variable?